### PR TITLE
shelters html and routes

### PR DESCRIPTION
**Pull request by ctedtsen**
I added the html pages and routes for the following in shelters:
/ route shows a list of all shelters and provides a link to each specific shelter
/:id route shows an html page for a specific shelter
/edit route shows a page with a form for adding and removing shelters

run npm start inside /src

### CODE REVIEW

**Comment by Jasmine211** [comment]
Hey, I checked if the html code was valid using https://validator.w3.org/nu/#textarea and there were a couple of errors on some pages (missing title and lang attribute ). Also, I think it would be better if you used the dl, dt, and dd tags instead of p since we are defining items in each shelter. Let me know if you have any questions :)

**Comment by ctedtsen** [comment]
Good catch, thanks! I think I've fixed the html issues and also changed the p tags to dl, dt, and dd tags.

File: src/app.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
+const path = require('path');
 const express = require('express');
+var session = require('express-session')
```
Comment (Jasmine211) [inline src/app.js]: We can keep this if you add express-session as a dependency on the package.json file, else remove.

File: src/helpers.js [hunk 2]
```diff
@@ -1,8 +1,9 @@
 const { objectId } = require("mongodb");
+const bcrypt = require('bcrypt');
```
Comment (Jasmine211) [inline src/helpers.js]: Need to add bcrypt dependency to package.json

**Review by Jasmine211 — CHANGES_REQUESTED** [review]
Hey, your html pages look good, but please see the comments I made for the app and helpers file

**Review by Jasmine211 — APPROVED** [review]

**Pull request merged**